client/.eslintrc.js

```diff
@@ -23,13 +23,17 @@ module.exports = {
     "no-lonely-if": "off",
     "consistent-return": "off",
     "no-control-regex": "off",
+    "no-script-url": "off", // some <a> tags should have href="javascript:void(0)"
     "react/jsx-filename-extension": "off",
     "react/jsx-uses-react": "error",
     "react/jsx-uses-vars": "error",
     "react/prefer-stateless-function": "warn",
     "react/forbid-prop-types": "warn",
     "react/prop-types": "warn",
     "jsx-a11y/anchor-is-valid": "off",
+    "jsx-a11y/click-events-have-key-events": "off",
+    "jsx-a11y/label-has-for": "off",
+    "jsx-a11y/no-static-element-interactions": "off",
```

Comment:
This one has an easy fix (add `role` value), so maybe we should keep it `on`? 

While it's probably a long way to make Redash fully accessible, let's try to minimize the future effort :)

Reply:
Right now I cannot remember why I decided to add this, but I remember that I did it after fighting for some time with all that crap. When adding `role="button"` to `<a>`, eslint stops showing this warning, but starts showing something else.